### DISCUSSION TITLE
qt.quickapp 如何启用 qmltc 和 qmlsc ?

### DISCUSSION THREAD

**Discussion by mengxinxiaoqiang** [xmake-io/xmake#4244] [Q&A] [answered]
![image](https://github.com/xmake-io/xmake/assets/56524631/026c49fa-2f4d-49cf-a56c-c14a8fac4c93)

**Comment by waruqi** [comment]
? 启用后做什么呢? 编译什么文件

**Reply by mengxinxiaoqiang** [reply]
启用后可以把QML 编译为CPP，提高效率

**Reply by waruqi** [reply, marked as answer]
目前不支持，可以提 feature request 过来，然后等着。。或者直接提 pr 过来
